noty-android/data/local/schemas/dev.shreyaspatil.noty.data.local.NotyDatabase/1.json

```diff
@@ -2,11 +2,11 @@
   "formatVersion": 1,
   "database": {
     "version": 1,
```

Comment:
You should change the DB version and write migration for adding `isPinned` column (_which you already have written_) in our DB. Otherwise, this will break on updating the app from the previous version.

So revert this file, and change DB version to 2, and rebuild project. Then checkout `2.json` file in VCS.